### DISCUSSION TITLE
lint script throwing an error:  "Plugin "react" was conflicted between [...]"

### DISCUSSION THREAD

**Discussion by ubmit** [vercel/next.js#32654] [Help]
`node` version: 16.2.0
`next` version: 12.0.4
`eslint` version: 7.32.0
`eslint-config-next` version: 12.0.4

Error message:

```sh
Plugin "react" was conflicted between ".eslintrc.json » eslint-config-next/core-web-vitals » /Users/guilherme/workspace/open-source/next.js/examples/with-vanilla-extract/node_modules/eslint-config-next/index.js" and "../../.eslintrc.json".
```

![image](https://user-images.githubusercontent.com/8797405/146673513-66d2f9a8-464e-4765-b4f9-6d00ea328337.png)

Draft PR: https://github.com/vercel/next.js/pull/32653

I went to the [documentation](https://nextjs.org/docs/basic-features/eslint#recommended-plugin-ruleset) but I couldn't find the answer there. Could someone help me on this, please? 🙏

**Comment by amine-abdelli** [comment]
Same problem here, did you find a solution?

**Reply by ubmit** [reply]
I didn't :(

**Comment by joaopedrodcf** [comment]
I'm having exactly the same problem but I'm pretty sure this was working before 😕

**Comment by ubmit** [comment]
Somehow this didn't happen when I created a Next app (using the same versions) outside of my fork. Still clueless though.

**Comment by HelaGone** [comment]
nine month after.. Did you guys found a solution, I'm having the same issue.

**Reply by ubmit** [reply]
I haven't
